packages/agents/src/orchestrator/index.ts

```
@@ -528,6 +535,40 @@ export class OrchestratorAgent extends BaseAgent {
+    if (workflowType === 'infra-setup' || workflowType === 'infra-deploy') {
+      switch (agentName) {
+        case 'triage':
+          // Triage expects { description, context }
+          return {
+            infraTask: typedInput.description || typedInput,
+            context: typedInput.context,
+          };
+
+        case 'infra':
+          // Infra expects { description, taskType, environment, ... }
+          const triageOutput = typedInput as { overallComplexity?: number };
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)